### DISCUSSION TITLE
Empty nuxt project has high LCP and FCP value in lighthouse

### DISCUSSION THREAD

**Discussion by siporoz** [nuxt/nuxt#28460] [Questions]
Hi,
I am faced with the problem that my project has high LCP and FCP values in lighthouse, even if there is a minimum number of elements on the page.
As an experiment, I decided to create a new empty project through the command:

`npx nuxi@latest init my-awsome-project`

Next, I built the project and launched it. 

And here are the values I got:

<img width="1202" alt="Снимок экрана 2024-08-08 в 10 58 23" src="https://github.com/user-attachments/assets/f4d06d7c-8d29-47f4-ac7a-fcd3ed2b8c0c">

Even if change `<NuxtWelcome />` e.g.` <h1>,` the values remain high for an empty page.

<img width="1391" alt="Снимок экрана 2024-08-08 в 11 01 19" src="https://github.com/user-attachments/assets/35eb83ad-453b-4c65-b5e7-413c3a606320">

### Environment

Operating System: mac os 14.4.1
Node Version: v22.1.0
Nuxt Version: 3.12.4

**Comment by lceihen** [comment]
This problem does exist, and as the elements on the page are added, the score will get worse, and some will even be reduced by 15 points.

**Comment by TheAlexLichter** [comment]
Please make sure to not measure in dev mode but after building.
